mlflow/models/docker_utils.py

```diff
@@ -152,6 +152,41 @@ def _get_mlflow_install_step(dockerfile_context_dir, mlflow_home):
         ).format(version=mlflow.version.VERSION, maven_proxy=maven_proxy)
 
 
+def _generate_dockerfile_raw(
```

Comment:
```suggestion
def _generate_dockerfile_content(
```

Can we rename this function `_generate_dockerfile_content`?